package.json

```diff
@@ -55,10 +56,14 @@
     "eslint-plugin-prettier": "^4.2.1",
     "jest": "29.3.1",
     "jest-environment-jsdom": "29.4.2",
+    "kleur": "4.1.5",
     "prettier": "^2.7.1",
+    "prompts": "2.4.2",
+    "semver": "7.3.8",
     "ts-jest": "29.0.1",
     "typescript": "4.9.4",
     "vite": "3.2.3",
-    "vite-plugin-dts": "1.7.1"
+    "vite-plugin-dts": "1.7.1",
+    "zx": "7.2.1"
```

Comment:
I've tried `zx` here. It's similar to `execa`, which helps running shell commands.

It may seem duplicated, but I just wanted to give it a try :)
https://github.com/google/zx

I can totally revert this back to using execa if you want.